package.json

```diff
@@ -40,8 +40,8 @@
     "mini-css-extract-plugin": "^0.11.0",
     "null-loader": "^3.0.0",
     "po2json": "^1.0.0-alpha",
-    "sass-loader": "^8.0.2",
-    "sass-resources-loader": "^2.0.3",
+    "sass": "~1.32.0",
```

Comment:
If this is a straight-up PF4 project, you might try relaxing the version restriction here.  Garrett mentioned something that PF4 is now Dart Sass compatible.

Reply:
I tried and there are some deprecation warnings.